org.eclipse.jdt.ui.tests.refactoring/test cases/org/eclipse/jdt/ui/tests/refactoring/ExtractMultipleTempTest1.java

```diff
@@ -0,0 +1,81 @@
+/*******************************************************************************
```

Comment:
This test has no way of running.  See other ExtractTemp tests for how to make these part of the automatic testsuite.

Reply:
will do in a minute.